### ISSUE TITLE
Search Results - WCAG 2.0 Level A Issue: 3.3.2 Labels or Instructions

### ISSUE THREAD

**Issue by rlee2015** [Sitefinity/feather-widgets#976]
Search Results - WCAG 2.0 Level A Issue: 3.3.2 Labels or Instructions

Please change line #25 and #26 in SearchResults.cshtml to:

&lt;label for="sortby"&gt;@Html.Resource("SortBy")&lt;/label&gt;
&lt;select class="userSortDropdown" id="sortby"&gt;

**Comment by PepiIvanova** [comment]
Hi, 

I am logging this as a bug that will be fixed in one of the upcoming releases.

Best,
Pepi

**Comment by mariushm** [comment]
Still getting error messages after using the fixed version. Here are the error messages from TotalValidator tool.

[WCAG2 1.1.1/1.3.1/3.3.2 (A)] Form controls without a label should have a 'title' attribute:
Use the 'title' attribute to label form controls when the visual design cannot accommodate the label (for example, if there is no text on the screen that can be identified as a label) or where it might be confusing to display a label. User agents, including assisting technology, can speak the 'title' attribute. See http://www.w3.org/TR/WCAG20-TECHS/H65.html (displayed in new window)

(posted on behalf of @rlee2015 )
